### DISCUSSION TITLE
现在这个版本用不了，搜索漫画直接就显示搜索错误

### DISCUSSION THREAD

**Discussion by lkjmnbvcx11** [lanyeeee/copymanga-downloader#9] [General]
如标题所说，反馈回来是这个：0: Request failed after 1 retries
1: error sending request for url (https://api.mangacopy.com/api/v3/search/comic?limit=20&offset=0&platform=4&q=%E9%AB%98%E8%BE%BE&q_type=)
2: operation timed out

**Comment by lanyeeee** [comment]
挂代理
